.github/workflows/backend.yml

```diff
@@ -99,22 +99,9 @@ jobs:
     - name: Build static viz frontend
       run: yarn build-static-viz
 
-    - name: Compile Java source file(s)
-      run: clojure -X:deps prep
-    - name: Compile driver AOT namespaces
-      working-directory: modules/drivers
-      run: clojure -X:deps prep
-    - name: Fetch dependencies
-      run: clojure -P -X:dev:ci:ee:ee-dev:drivers:drivers-dev
-    - name: Fetch dependencies  (./bin/build/build-mb)
-      working-directory: bin/build-mb
-      run: clojure -P -M:test
-    - name: Fetch dependencies (./bin/build/build-drivers)
-      working-directory: bin/build-drivers
-      run: clojure -P -M:test
```

Comment:
None of this stuff is needed before running backend unit tests, so not only does it potentially slow things down, it might also potentially break unrelated things e.g. unit tests can fail if there is an error in the build scripts `deps.edn`